camayoc/utils.py

```diff
@@ -26,6 +26,11 @@
 client_cmd = settings.quipucords_cli.executable
 """Client command to use during tests. Defaults to `qpc`."""
 
+client_cmd_name = settings.quipucords_cli.display_name
```

Comment:
I'm not intimately familiar with all these use-cases, but couldn't we compute that automatically?
Something like:

```
client_cmd_name = pathlib.Path(settings.quipucords_cli.executable).name
```

Reply:
I thought about that, but it wouldn't cover wild use cases like
`qpc -v` or `toolbox run --container discovery-toolbox dsc`

Reply:
I'm not sure if quipucords_cli.executable should ever be set to `qpc -v` - after all, testing verbose mode is legitimate, so that should be up to specific test to decide if it wants additional flags or not.

tollbox makes more sense. I imagine that there should be a list of transparent wrappers somewhere, and toolbox could be one of them. Then when you test locally you just call the executable, when you test upstream on Jenkins executable would be wrapped in SSH (to connect to remote host) and when you test downstream on Jenkins, executable would be wrapped in SSH *and* toolbox.